Examples/UICatalog/Scenarios/Buttons.cs

```diff
@@ -138,7 +138,8 @@ static void DoMessage (Button button, string txt)
         var removeButton = new Button
         {
             X = 2, Y = Pos.Bottom (button) + 1,
-            ColorScheme = Colors.ColorSchemes ["Error"], Text = "Remove this button"
+            SchemeName = "Error",
```

Comment:
So string is good for this,  but we should use a central const for the known core ones

Reply:
```diff
+ SchemeName = SchemeManager.SchemesToSchemeName(Schemes.Error),
```